components/bl-signature-pad-component/src/buttons/save-png-button.js

```diff
@@ -0,0 +1,32 @@
+import download from '../utils/download';
+
+export default function SavePNGButton(props) {
+  const { signaturePad, component, eventHandlers } = props;
+
+  const savePNG = () => {
+    if (signaturePad.isEmpty()) {
+      alert('Please provide a signature first.');
+    } else {
+      const dataURL = signaturePad.toDataURL();
+
+      download(dataURL, 'signature.png');
+    }
+
+    if (eventHandlers.onSaveClick) {
+      eventHandlers.onSaveClick();
+    }
+  };
+
+  return (
+    <button
+      style={{
+        display: component.savePNGButtonVisibility === 'true' ? 'inline-block' : 'none',
+      }}
+      onClick={ savePNG }
+    >
+      {/*SHOULD BE
```

Comment:
Will be better if we leave a ticket number at comment like this
/* Waiting for BKNDLSS-28470
...<how should be>... */